src/3d/symbols/qgspointcloud3dsymbol.h

```diff
@@ -29,48 +33,191 @@
  *
  * \since QGIS 3.18
  */
-class _3D_EXPORT QgsPointCloud3DSymbol : public QgsAbstract3DSymbol
+class _3D_EXPORT QgsPointCloud3DSymbol : public QgsAbstract3DSymbol SIP_ABSTRACT
 {
   public:
+
+    /**
+     * How to render the point cloud
+     */
+    enum RenderingStyle
+    {
+      // Do not render anything
+      NoRendering = 0,
+      //! Render the point cloud with a single color
+      SingleColor,
+      //! Render the point cloud with a color ramp
+      ColorRamp,
+      //! Render the RGB colors of the point cloud
+      RGBRendering
```

Comment:
Please rename to `RgbRendering` for consistency (in acronyms we only keep the first letter upper case)